app/controllers/api/v1/admin/project.js

```diff
@@ -2,6 +2,12 @@ const { ProjectStatus } = global.appEnums;
 const moment = require('moment');
 const { sumBy, flatMap, map, filter } = require("lodash");
```

Comment:
why we require lodash 2 times in a same file?

Reply:
It was included by mistake while resolving the conflicts.